lib/index.js

```diff
@@ -183,7 +185,8 @@ api.insert = brCallbackify(async ({actor, account, meta}) => {
       'DuplicateError', {
         public: true,
         httpStatusCode: 409
-      }, e);
+      }, e
+    );
```

Comment:
Needs fixing with proper linter rule that never puts parenthesis on its own line by itself.